pkgs/top-level/all-packages.nix

```
@@ -32838,6 +32838,8 @@ with pkgs;
   endless-sky = callPackage ../games/endless-sky { };
 
   enyo-doom = libsForQt5.callPackage ../games/enyo-doom { };
```

Should this line not be changed to enyo-launcher?

---

I was about to say, ~~it seems as though~~ upstream simply renamed the project. We might want to remove and alias the old derivation then. @tadfisher what do you think?

---

```suggestion
  enyo-doom = libsForQt5.callPackage ../games/enyo-launcher { };
```
I mean, unless there is a fork between enyo-launcer and enyo-doom, and I would imagine removing ../games/enyo-doom would also be part of the change.

If there is a facility for deprecating packages that would probably also be useful

---

FWIW, I'd seen some discussion about removing packages, but this is potentially the replacement.  For whatever reason upstream has both an enyo-doom *and*an enyo-launcher git repository.

---

The manual says that there's no policy. IMHO it seems a bit pointless to keep both versions side-by-side just because there exists an archived repository on gitlab. I don't know the software but judging by versioning conventions 2.0.5 should be at least as good as 2.0.2. Feel free to correct me.

---

It is the same upstream, the project was renamed. We can rename the attribute here, just add an alias in `aliases.nix` so as to not break user configs.

---

I seem to have wandered in over my head, so I'll ask what to do before I push anything:

I'll remove the enyo-doom folder, as enyo-launcher is replacing it and add an alias to pkgs/toplevel/aliases.nix

I would assume, based on what exists, the alias would be:

```enyo-doom: throw "enyo-doom has been renamed to/replaced by enyo-launcher"; # Added 9/9/2022```, and I assume amend the commit that adds enyo-launcher commit to integrate the deletion/alias?

Would I need some way to link enyo-doom to enyo-launcher, or is that it?

---

Imho it's nicer and simpler to do `enyo-doom = enyo-launcher; # Added 9/9/2022`. Otherwise you nailed it